glacier_mapping/models/frame.py

```diff
@@ -144,7 +186,7 @@ def metrics(self, y_hat, y, metrics_opts):
             if "threshold" in metric.keys():
                 y_hat = y_hat > metric["threshold"]
 
-                metric_fun = globals()[k]
-                metric_value = metric_fun(y_hat, y)
-            results[k] = np.mean(np.asarray(metric_value))
+            metric_fun = globals()[k]
+            metric_value = metric_fun(y_hat, y)
```

Comment:
is `metric_value` used again later? if not, just use the output of the function directly in `results[k]`